core/modules/qmeta/QMetaMysql.cc

```diff
@@ -296,9 +297,9 @@ QMetaMysql::registerQuery(QInfo const& qInfo,
         query += _conn.escapeString(itr->second);
         query += "')";
 
-        LOGS(_log, LOG_LVL_DEBUG, "Executing query: " << query);
+        LOGS(_log, LOG_LVL_DEBUG, qIdStr << " Executing query: " << query);
```

Comment:
I'm not sure you want query ID in this message, this will be printed even before "UserQuery registered" message in UserQuerySelect and I thought you insisted on "UserQuery registered" being the very first message with query ID.